django_saml2_auth/views.py

```diff
@@ -106,10 +106,23 @@ def _get_saml_client(domain, metadata_conf_url):
     # > application. We wanted to provide SAML for our customers, so we used
     # > this as a way to do that.
     #
-    settings.SAML2_AUTH['METADATA_AUTO_CONF_URL'] = metadata_conf_url
+    # We are also stopping using the _get_metadata function altogether.
+    # Since our support is for SAML 2.0 and not a specific company, changing
+    # the settings object everytime could lead to issues due to concurrency.
+    # For now, we won't be supporting the LOCAL_FILE_PATH setting.
+    # Related:
+    # https://github.com/Tesorio/django-saml2-auth/pull/11#pullrequestreview-704613069
+    # We will give priority to the raw XML file if it exist
+    if metadata_conf_raw:
+        metadata = {'inline': [metadata_conf_raw]}
+    else:
+        metadata = {
+            'remote': [
+                {'url': metadata_conf_url},
+            ]
+        }
 # END TESORIO CHANGES
     acs_url = domain + get_reverse([acs, 'acs', 'django_saml2_auth:acs'])
-    metadata = _get_metadata()
```

Comment:
These deletions are Tesorio changes. 
Instead of deleting these lines, I would suggest commenting them out and moving the `# END TESORIO CHANGES` comment below to encapsulate the entire block of changes.

When reviewing this library, I found this helpful to understand the changes made. See examples on lines:
* 98
* 195
* 301

I know there is blame history but I think we should be consistent.

Reply:
Comments added, thanks for point this out